app/services/reports/period_supply_report_service.rb

```diff
@@ -14,8 +14,8 @@ def initialize(year:, organization:)
     def report
       @report ||= {name: "Period Supplies",
                    entries: {
-                     "Period supplies distributed" => number_with_delimiter(distributed_supplies),
-                     "Period supplies per adult per month" => monthly_supplies&.round || 0,
+                     "Period supplies distributed" => number_with_delimiter(total_distributed_period_supplies),
+                     "Period supplies per adult per month" => (monthly_supplies || 0 + distributed_period_supplies_from_kits_per_month)&.round,
```

Comment:
This is just going to give you monthly_supplies.round, if you have any.  I'm not sure why we had the "|| 0" in the original